### ISSUE TITLE
The estimator object of the final step (Python class sklearn.svm._classes.SVR) does not specify the number of outputs

### ISSUE THREAD

**Issue by mxsrv** [jpmml/sklearn2pmml#352]
Hey,

I am trtying to convert a sklearn SVR model to PMML and currently facing this error:

```
Standard output is empty
Standard error:
Exception in thread "main" java.lang.IllegalArgumentException: The estimator object of the final step (Python class sklearn.svm._classes.SVR) does not specify the number of outputs
	at sklearn2pmml.pipeline.PMMLPipeline.initTargetFields(PMMLPipeline.java:564)
	at sklearn2pmml.pipeline.PMMLPipeline.encodePMML(PMMLPipeline.java:132)
	at com.sklearn2pmml.Main.run(Main.java:91)
	at com.sklearn2pmml.Main.main(Main.java:66)  
```

This is my pipeline:

```python

from sklearn2pmml import PMMLPipeline, sklearn2pmml
from sklearn.svm import SVR

model2 = SVR(kernel="poly", C=1750.0, gamma="scale",
                        degree=3, epsilon=0.1, coef0=1)

modelSave = PMMLPipeline([('regression', model2)])
```

Am I missing something?

**Comment by vruusmann** [comment]
> Am I missing something?

You are currently missing a `(PMML)Pipeline.fit(X, y)` method call:

```python
modelSave.fit(X, y)
```

This error means that JPMML-SkLearn cannot figure out how many target variables there are involved. In most cases it would be safe to assume 1 (ie. a single-label ML exercise), but sometimes its >1 (ie. multi-label ML exercises).

The `PMMLPipeline` class collects this information (and many other relevant pieces of information such as feature and target names, etc) during `PMMLPipeline.fit(X, y)`.

Alternatively, you may provide this information manually by setting the `PMMLPipeline.target_fields` attribute:
```python
modelSave.target_fields = ["my_single_target"]
```

**Comment by mxsrv** [comment]
Specifiying target field solved the issue. Big thanks!
